src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -198,7 +197,8 @@ private void burnFlash() {
    */
   private void setSpeed(SwerveModuleState desiredState, boolean isOpenLoop) {
     if (isOpenLoop) {
-      double percentOutput = desiredState.speedMetersPerSecond / Config.Swerve.maxSpeed;
```

Comment:
Leave this line and comment it out.
Write obvious notes there. For our future testing.